### PR TITLE
Testnet guide copy edits

### PR DESCRIPTION
**Pull request by lrettig**
Begin to tweak the copy

### CODE REVIEW

File: wallet.md [hunk 1]
```diff
@@ -54,41 +50,36 @@ Click the `ALL TRANSACTIONS` button to access your wallet transactions log scree
 Follow [this guide](wallet_logs.md) to learn how to work with the transactions log screen.
 
 ## Sending Smesh
-Follow the steps in [this guide](send_coin.md) to send Smesh to any other account.
+Follow the steps in [this guide](send_coin.md) to send SMH to another account.
```
Comment (avive) [inline wallet.md]: Smesh is more general. You actually currently send Smidge...

File: dict.md [hunk 2]
```diff
@@ -1,49 +1,48 @@
...
+Your Smesher is responsible for submitting blocks with Smesh transactions to the network according to the Spacemesh protocol. We call this Smeshing. You earn Smeshing rewards for Smeshing.
 
-## Free Disk Space Commitment
-Explain here...
+The Spacemesh App for your desktop computer includes a Smesher and manages it for you. Future releases of the Testnet will support building and running a Smesher directly from source code.
 
 ## Smeshing
-Smeshing is the process of submitting blocks with transactions to the Testnet network according to the Spacemesh protocol. Before you can smesh blocks, you need to commit XXXX free storage on your computer hard drive to Spacemesh. This commitment is a one-time process that you only need to do once. You can use the disk space you committed to Spacemesh indefinitely.
+Smeshing is the process of submitting blocks with transactions to the Testnet network according to the Spacemesh protocol. Before you can Smesh blocks, you need to commit 4GB free storage on your computer hard drive to Spacemesh (for the Open Testnet; this amount will increase to 100GB for the Mainnet).
```
Comment (avive) [inline dict.md]: Likely 100GB in the testnet at some point once GPU smeshing is implemented.

File: get_coin.md [hunk 3]
```diff
@@ -1,48 +1,45 @@
...
+3. Send the copied address to a friend to request that they send you SMH. You can paste the address directly into an app such as Whatsapp, Signal, or Telegram.
 
-4. Ask that person to send Smesh to this address using his or her Spacemesh wallet.
+4. Ask that person to [send SMH](send_coin.md) to this address using his or her Spacemesh wallet.
 
-> Once Smesh are sent to you, you should see the transaction in the `Transactions` section of your wallet screen, and your account Smesh balance should be updated to include the received Smesh.
+> Once Smesh are sent to you, you should see the transaction in the `Transactions` section of your wallet screen, and your account Smesh balance should be updated to include the received SMH.
 
-Congratulations! you have successfully received Smesh from another account to your account!
+Congratulations! You have successfully received your first SMH tokens!
```
Comment (avive) [inline get_coin.md]: We don't use token anywhere - SMH coins

File: guide/setup.md [hunk 4]
```diff
@@ -12,17 +12,17 @@ Let's now proceed to set up the Smesher, Smeshing, and the Wallet. But first, so
 
 ## About the Wallet
 
-A wallet holds one or more Smesh accounts. An account is identified by a long, unique number, such as `0x20a5...3a1f`.
+A Wallet holds one or more Smesh accounts. An account is identified by a long, unique hexidecimal number, such as `0x20a5...3a1f`.
 
-The wallet enables you to send and receive Smesh, and in the future, will enable you to run decentralized applications. Each account has a Smesh balance.
+The Wallet enables you to send and receive Smesh, and in the future, it will enable you to run decentralized applications (dapps) built on Spacemesh. Each account has a balance of Smesh tokens (SMH).
```
Comment (avive) [inline guide/setup.md]: tokens -> coins

File: get_coin.md [hunk 5]
```diff
@@ -1,48 +1,45 @@
 # Getting Testnet Smesh
 
-There are 3 ways to get Testnet Smesh:
-1. Earn smeshing rewards. Follow [this guide](rewards.md) to set up smeshing.
-2. Get Smesh from the [Spacemesh Tap](https://discord.gg/ASpy52C).
-3. Get Smesh sent from another account to your account in a transaction.
+There are 3 ways to get Testnet Smesh (SMH) tokens:
```
Comment (avive) [inline get_coin.md]: Smesh tokens -> Smesh coins

File: main.md [hunk 6]
```diff
@@ -2,66 +2,66 @@
...
 
 ## Why should I join the Spacemesh Testnet?
-Joining the Testnet helps Spacemesh launch a high-quality Mainnet and the `Smesh` (SMH) cryptocurrency. Spacemesh is a decentralized ledger designed to be formed from a large number of home desktop PC owners who run the Spacemesh software on their PCs. To test such a network we need a large number of block producers to join the Testnet.
+Joining the Spacemesh Open Testnet today gives you an early opportunity to participate in the most open, most permissionless, fairest blockchain and cryptocurrency project in the world. It gives you the opportunity to mine from home, using existing hardware, without needing any special tooling, cloud services, or devops tools. Joining the Testnet today will give you a leg up on joining the Spacemesh Mainnet when it launches and mining Smesh tokens from home. It also gives you a chance to contribute to the project and to the code, since everything is open source!
```
Comment (avive) [inline main.md]: tokens -> coins

File: send_coin.md [hunk 7]
```diff
@@ -10,11 +10,11 @@ You should see the Send SMH screen:
...
 ![](images/v1.0/send_smc_tx.png)
 
-2. Paste the Smesh recipient public account address that he or she shared with you into the `Send to` field.
+2. Paste the recipient's account address that he or she shared with you into the `Send to` field.
 
-3. Enter the Smesh (SMH) amount you would like to send to the recipient in the `Amount to send` field.
+3. Enter the Smesh (SMH) amount you would like to send to the recipient in the `Amount` field.
 
-> The amount must be less than your account's Smesh balance, as you cannot spend Smesh that you do not have.
+> The amount must be less than your account's SMH balance, as you cannot spend tokens that you do not have.
```
Comment (avive) [inline send_coin.md]: tokens -> coins

File: wallet.md [hunk 8]
```diff
@@ -1,51 +1,47 @@
...
-The left side of the screen displays the wallet info:
+The left side of the screen displays the Wallet info:
 
 ![](images/v1.0/wallet_details.png ':size=310x496')
 
-Your wallet is configured with one main account called `Main Account`.
+Your Wallet is configured with one main account called `Main Account`.
 
-Think of this account as your crypto bank account. You use this account to hold Smesh and to send and receive Smesh.
+Think of this account as your crypto bank account. You use this account to hold SMH and to send and receive SMH tokens.
```
Comment (avive) [inline wallet.md]: SMH tokens -> Smesh/SMH coins

File: wallet.md (hunk 1 above)
Comment (lrettig) [reply wallet.md]: This terminology is very confusing. "Smesh" and "SMH" should be the same. It's bad enough that "Smesh" also refers to "Smeshing." I'll replace with something more generic like "coin."

(For the record, in Ethereum, it's called "sending ETH" even if I'm only sending a few wei.)

Comment (avive) [reply wallet.md]: So let's keep send SMH for now. We'll revisit coin names for sm 0.2 as planned.

**Pull request merged**